app.py

```diff
@@ -2,11 +2,18 @@
 
 from dotenv import load_dotenv
 from flask import Flask, g, jsonify, request
-from flask_aserto import AsertoMiddleware, AuthorizationError, ResourceContext
+from flask_aserto import AsertoMiddleware, AuthorizationError
+from aserto.client import ResourceContext
```

Comment:
Why are you changing this import to come from `aserto.client` instead of `flask_aserto`? People who use flask_aserto shouldn't have to know that under the hood ResourceContext is defined in aserto.client. They should be able to do everything they need by importing types from flask_aserto.